common/diplo_phrases/hfe_diplo_phrases.txt

```diff
@@ -94,7 +94,10 @@ greetings = {
 		trigger = {
 			has_country_flag = hfe_egypt
 			from = {
-				has_ascension_perk = ap_the_flesh_is_weak
+				OR = {
```

Comment:
this diplo phrase refer to perk, while you can get cybernetic pop from assimilator, for example

Reply:
Okay, I'll change it at the end.

Reply:
Okay, done